### DISCUSSION TITLE
posts/%E8%87%AA%E7%A0%94%E4%BF%A1%E6%81%AF%E6%94%B6%E9%9B%86%E5%B7%A5%E5%85%B7%E5%88%86%E4%BA%AB/

### DISCUSSION THREAD

**Discussion by giscus** [z2p/z2p.github.io#2] [Q&A]
# posts/%E8%87%AA%E7%A0%94%E4%BF%A1%E6%81%AF%E6%94%B6%E9%9B%86%E5%B7%A5%E5%85%B7%E5%88%86%E4%BA%AB/

0x01 前言 - sweetPotato的由来

https://z2p.github.io/posts/%E8%87%AA%E7%A0%94%E4%BF%A1%E6%81%AF%E6%94%B6%E9%9B%86%E5%B7%A5%E5%85%B7%E5%88%86%E4%BA%AB/

**Comment by z2p** [comment]
Hello！

**Comment by Zshan7que** [comment]
老哥，我这里一直显示加载config.json错误

**Reply by z2p** [reply]
可以试试更新一下java版本，测试过主要是版本不一致会出现这类型的情况。第二个可能是，修改过config.json，有json语法错误，比如多了一个逗号之类的

**Comment by Dayu009** [comment]
老哥，怎么一直报错java.lang.Exception: Json½âÎö³öÏÖÒì³££ºD:\tools\burpcj\sweetPotato1.6\resources\finger.json Cannot invoke "java.util.List.add(Object)" because "list" is null

**Comment by shannaburns** [comment]
java11报错ava.lang.Exception: Json½âÎö³öÏÖÒì³££ºD:\tools\burpcj\sweetPotato1.6\resources\finger.json Cannot invoke "java.util.List.add(Object)" because "list" is null ，怎么都不好使

**Comment by rzte** [comment]
不错不错
